sass/_SmartSnippet.scss

```diff
@@ -168,8 +168,8 @@
           }
 
           &.coveo-user-feedback-banner-yes-button {
-            color: $color-green;
-            fill: $color-green;
+            color: $color-strong-contrast-green;
```

Comment:
it has a strong contrast on white, not necessarily all the time, would `$color-dark-green` be a better name?

Reply:
Good point, I'll rename it.